python-checks/src/test/resources/checks/identityComparisonWithCachedTypes.py

```diff
@@ -160,3 +160,10 @@ def potential_null_symbols():
 def comparison_to_class(arg):
```

Comment:
Note on PR title/ticket name: they're not FPs strictly speaking. They just might be a bit noisy for users.
I think you could say "Document S5795 new issues stemming from better type resolution" to be more neutral.